src/types/chartdata.raw.ts

```diff
@@ -5,7 +5,9 @@ export const ChartdataV1 = z.object({
   voteEnd: z.number(),
   totalVotes: z.number(),
   bribedVotes: z.number(),
+  externallyBribedVotes: z.number(),
   totalBribes: z.number(),
+  totalExternalBribes: z.number(),
```

Comment:
please make it '.optional()' to avoid conflicts with old data, that doesn't have this data